bitmovin-analytics-datasource/src/components/GroupByInput.tsx

```diff
@@ -26,7 +26,7 @@ export function GroupByInput(props: Props) {
     <HorizontalGroup>
       <Select
         value={isEmpty(props.groupBy) ? undefined : props.groupBy}
-        onChange={props.onChange}
+        onChange={(value) => props.onChange(value.value!)}
```

Comment:
```suggestion
        onChange={(selectableValue) => props.onChange(selectableValue.value!)}
```

and on another places same